webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -44,26 +44,28 @@ const TrialChart: React.FC<Props> = ({
         const y = metric.type === MetricType.Validation ?
           validationSource[metric.name] : trainingSource[metric.name];
 
+        const metricKey = metric.type+'_'+metric.name;
```

Comment:
**nit:** Super minor thing, but we try to use template literals whenever possible.
```
const metricKey = `${metric.type}_${metric.name}`;
```

Reply:
Thanks for the review. Changed.